From b925e25de35dd4fbf93ab6688284dd5a43532e71 Mon Sep 17 00:00:00 2001
From: Jorgen Schaefer <Jorgen.Schaefer@mindmatters.de>
Date: Tue, 31 May 2016 16:08:49 +0200
Subject: [PATCH 3/3] Mindmatters-Config integrieren

---
 bash.sh | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/bash.sh b/bash.sh
index 3c04db2..0e0cc20 100755
--- a/bash.sh
+++ b/bash.sh
@@ -74,6 +74,22 @@ then
     export PATH="$GOPATH/bin:$PATH"
 fi
 
+if [ -d "$HOME/.nvm" ]
+then
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+fi
+
+if [ -d "$HOME/.rbenv" ]
+then
+    export PATH="$HOME/.rbenv:$PATH"
+fi
+
+if [ -d "/usr/lib/chromium" ]
+then
+    export PATH="/usr/lib/chromium:$PATH"
+fi
+
 # Only in interactive shells
 if [ -n "$PS1" ]
 then
-- 
2.8.1

